front/app/containers/Admin/projects/project/nativeSurvey/FormResults/analysis/AnalysisInsights.tsx

```diff
@@ -36,7 +41,12 @@ const Summary = ({
   }
   return (
     <Text fontSize="s" mt="0px">
-      {removeRefs(summary)}
+      {replaceIdRefsWithLinks({
```

Comment:
Is this the view in the project/phase settings before you go to the analysis? What happens when you click on the refs in this case?

Reply:
The same thing happens in both cases - the analysis tool is opened and the input that corresponds to the reference is selected. When the user is already in the analysis, the pages don't change, of course, so the perception is that you are only previewing different inputs. To make it work both ways was the reason to add the selected input id to the url query parameters.